src/containers/VedtakContainer.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {
+    useEffect,
+    useState,
+} from 'react';
+import Side from '../sider/Side';
+import {
+    Column,
+    Row,
+} from 'nav-frontend-grid';
+import Panel from 'nav-frontend-paneler';
+import {
+    useDispatch,
+    useSelector,
+} from 'react-redux';
+import { hentVedtak } from '../actions/vedtak_actions';
+import {
+    Undertekst,
+    Undertittel,
+} from 'nav-frontend-typografi';
```

Comment:
Er det ikke noe lint-regel for: absolute imports before relative imports lenger?

Reply:
Virker som om det har endret seg.